tools/arcilator/arcilator.cpp

```diff
@@ -444,6 +450,8 @@ static LogicalResult processBuffer(
             llvm::errs() << "failed to create execution engine: "
                          << info.message() << "\n";
           });
+
+      arc_jit_runtime_env_deinit();
```

Comment:
nit: you can use` llvm::make_scope_exit` to automatically call `deinit`.

Reply:
Nice. Thanks!